docs/use-cases/create-module.md

```diff
@@ -997,12 +1100,12 @@ int main(int argc, char** argv) {
 };
 ```
 
-</details>
+{{< /expand >}}
 
 {{% /tab %}}
 {{< /tabs >}}
 
-### Configure logging
+#### (Optional) Configure logging
```

Comment:
I think we should also do what Bill suggested and add a step where you write code that works first (without viam sdk) and then you wrap it into a module

Reply:
^ I plan to do that step in some how-tos such as "How to write a sensor module with Python" but for this idk since it might not always be the way? Maybe instead, add it as a tip alert on this one?